packages/core/src/internal/fetchSource.ts

```diff
@@ -146,6 +146,14 @@ export const makeFetchSource = (
       })
       .then(complete)
       .catch((error: Error) => {
+        if (error.name === 'SyntaxError' || error.name === 'TypeError') {
```

Comment:
This might be a bit too limiting 😅

Reply:
Should we limit this to development?

Reply:
Well my thinking was that people using for instance Sentry in production would want to see these errors surface as well

Reply:
oh yea good point